### PR TITLE
Implement basic multiple docsets support

### PR DESCRIPTION
**Pull request by yufeih**
Ignore multiple docset on loc cases.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/5153)

### CODE REVIEW

File: src/docfx/lib/log/Log.cs [hunk 1]
```diff
@@ -32,6 +32,19 @@ public static void Important(string message, ConsoleColor color)
             }
         }
 
+        public static void Error(Error error, ErrorLevel? level = null)
```
Comment (live1206) [inline src/docfx/lib/log/Log.cs]: think `ConsoleLog` is better than `Error`

File: src/docfx/config/ConfigLoader.cs [hunk 2]
```diff
@@ -18,13 +19,41 @@ internal class ConfigLoader
...
         }
 
+        public static (string docsetPath, string outputPath)[] FindDocsets(string workingDirectory, CommandLineOptions options)
+        {
+            if (DisableMultipleDocsets != null && DisableMultipleDocsets())
+            {
+                return new[] { (workingDirectory, options.Output) };
+            }
+
+            return Directory.GetFiles(workingDirectory, "docfx.yml", SearchOption.AllDirectories)
```
Comment (live1206) [inline src/docfx/config/ConfigLoader.cs]: Doesn't docfx v3 accept `docfx.json` as well?
Or does this mean multiple-docset should only run after config migration tool for Docs?

Comment (yufeih) [reply src/docfx/config/ConfigLoader.cs]: yes, it should support `docfx.json`, see TODO comments in line 44

File: src/docfx/lib/log/Log.cs (hunk 1 above)
Comment (yufeih) [reply src/docfx/lib/log/Log.cs]: yep, initially I thought all console writes goes to `Log`, turned out to be not true

File: src/docfx/config/ConfigLoader.cs [hunk 3]
```diff
@@ -18,13 +19,41 @@ internal class ConfigLoader
...
+
+                // TODO: look for docfx.json after config migration tool has been merged info docfx
```
Comment (live1206) [inline src/docfx/config/ConfigLoader.cs]:
```suggestion
                // TODO: look for docfx.json after config migration tool generates docfx.yml the same directory as docfx.json
```

**Pull request merged**